dragonphy/tester.py

```diff
@@ -24,9 +24,12 @@ def __init__(self, top_module='test', ext_test_bench=True, disp_type='realtime',
 
 		# adjust defines
 		defines['DAVE_TIMEUNIT'] = '1fs'
-		defines['NCVLOG'] = None
+		if simulator == 'vcs':
```

Comment:
i added some experimental support for VCS.  i'm not yet convinced it is yielding accurate results, though.  i think we would need to go back to mLingua and check the Makefiles to make sure that I'm using all of the required flags here